src/accelerate/utils/imports.py

```diff
@@ -311,6 +320,26 @@ def get_major_and_minor_from_version(full_version):
     return True
 
 
+@lru_cache()
+def is_mlu_available(check_device=False):
+    "Checks if `torch_mlu` is installed and potentially if a MLU is in the environment"
+    if importlib.util.find_spec("torch") is None or importlib.util.find_spec("torch_mlu") is None:
```

Comment:
Accelerate doesn't work without torch, so we don't need the torch check. (I know it's similar to others, this is something we need to change eventually :) )